plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/address-line-2-field.tsx

```diff
@@ -9,14 +9,14 @@ import { __, sprintf } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { AddressFieldProps } from './types';
+import { AddressLineFieldProps } from './types';
 
-const Address2Field = < T extends AddressFormValues | ContactFormValues >( {
+const AddressLine2Field = < T extends AddressFormValues | ContactFormValues >( {
```

Comment:
I normalized `AddressField`, `AddressFields`, and `Address2Field` usage to AddressLine. cc @nielslange

Reply:
This is to reduce confusion with the address form and fields.

Reply:
Sounds good, @senadir. Thanks for improving this.